packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -297,4 +301,76 @@ class WKWebView {
   Future<void> loadRequest(NSUrlRequest request) {
     throw UnimplementedError();
   }
+
+  /// Loads the contents of the specified HTML string and navigates to it.
+  Future<void> loadHtmlString(String string, String? baseUrl) {
+    throw UnimplementedError();
```

Comment:
Nit: Maybe the PR should be called "Add stubs for WKWebView methods"? It's not actually implementing any of them.